addons/voice/org.openhab.voice.googletts/src/main/java/org/openhab/voice/googletts/internal/GoogleCloudAPI.java

```diff
@@ -105,13 +102,23 @@ void setConfig(GoogleTTSConfig config) {
                 voices.clear();
             }
         }
+
+        //maintain cache
+        if (config.getPurgeCache() != null && config.getPurgeCache()) {
+            File[] files = cacheFolder.listFiles();
+            if (files != null && files.length > 0) {
+                //noinspection ResultOfMethodCallIgnored
```

Comment:
Looks like these are some kind of IntelliJ specific annotations similar to `@SuppressWarnings`. I don't see them used anywhere else in the openHAB/ESH code base. Can you change your IDE settings to suppress the warnings and remove these comments?